lib/internal/Magento/Framework/File/Mime.php

```diff
@@ -80,10 +80,11 @@ class Mime
      * Get mime type of a file
      *
      * @param string $file
+     * @param string $default
      * @return string
      * @throws \InvalidArgumentException
      */
-    public function getMimeType($file)
+    public function getMimeType($file, $default = 'application/octet-stream')
```

Comment:
Hi @progreg , unfortunately, we can't change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Hi @sidolov, I've added changes according to Backward Compatible Development Guide, I hope all was done in the right way